README.md

```diff
@@ -95,5 +133,15 @@ Should you not be sure what to do about it (licensing produces severe headhaces)
  - [Choose a license](http://choosealicense.com/)
  - [tl;drLegal](https://tldrlegal.com/)
 
-## Other template and examples
+[`Go to the top`](#overview)
+
+# 💼 Other template and examples
 The [Awesome CMake](https://github.com/onqtam/awesome-cmake) repository contains an interesting [list of template and examples](https://github.com/onqtam/awesome-cmake#examples--templates) similar to this one.
+
+[`Go to the top`](#overview)
+
+
+---
+If you feel this CMake project template was useful, consider leaving the following shield somewhere in your README project (feel free to modify its look as you please). Otherwise, not a big deal! 👍
```

Comment:
If we are interested in dissemination, we can also ask for a GitHub star.

Reply:
Good point 👍 What do you suggest to do? Remove this comment and kindly ask for stars, or to also add star?